.github/workflows/linters-and-checks-linux.yml

```diff
@@ -0,0 +1,31 @@
+name: per-pr (linux)
```

Comment:
while we're in here, can we change the names of these tests to something a bit easier to read in the test matrix? every test starts with `per-pr` so they're kinda hard to tell apart. how about `Linters and Checks`?

Reply:
I'm fine w/ changing them.  The idea is that in the github UI, the name will stutter a bit with the naming suggested:
Current naming:
![image](https://user-images.githubusercontent.com/1951658/131415250-18bdd622-da95-411c-a9c5-29ead3aeb643.png)

New naming:
Linters and Checks / PR linters, checks, and unit tests

Reply:
I've opted for verbose names for both fields - `name:` and `build:` so we will have the stutter but it will be easier to tell tests apart in all UIs (Actions UI, etc.)